### ISSUE TITLE
Remove SonarCloud support

### ISSUE THREAD

**Issue by nathanklick** [hashgraph/hedera-services#11119]
## Description

Once the Codacy tooling has been onboarded, we need to remove the SonarCloud configuration and CI tasks.

### Related Issues

- Depends on #11117 
- Depends on #11118

**Comment by nathanklick** [comment]
Duplicate of #11574
